cognite/data_transfer_service.py

```
@@ -240,6 +240,28 @@ def __init__(
         self.cookies = cookies
         self.num_of_processes = num_of_processes
 
+    def get_time_series_name(self, ts_label: str = "default", dataframe_label: str = "default"):
+        if self.ts_data_specs is None:
+            raise InputError("Data spec does not contain any TimeSeriesDataSpecs")
+        if self.ts_data_specs is None:
+            raise InputError("Data spec does not contain any TimeSeriesDataSpecs")
+
+        tsds = None
+        for ts_data_spec in self.ts_data_specs:
+            if ts_data_spec.label == dataframe_label:
+                tsds = ts_data_spec
+
+        if tsds:
```

Should we raise an exception if not?